### ISSUE TITLE
RuntimeError: DataLoader worker (pid 22048) is killed by signal: Killed. 

### ISSUE THREAD

**Issue by gabays** [mittagessen/kraken#266]
I have a problem I cannot identified the source…
```
[gabays@gpu006 datasetsOCRSegmenter17]$ ketos train -t train.txt -e val.txt -f alto -d cuda

Building training set  [#################-------------------]  8030/16639  00:01:18[83.7327] Text line " " is empty after transformations 
Building training set  [####################################]  16639/16639          
Building validation set  [####################################]  1728/1728          [357.3623] alphabet mismatch: chars in training set only: {'k', 'É', '[', 'Ë', 'K', 'æ', 'À', 'ɔ', 'Ü', 'Ç', '–'} (not included in accuracy test during training) 
[357.3626] alphabet mismatch: chars in validation set only: {'Æ'} (not trained) 
Initializing model ✓
stage 1/∞  [------------------------------------]  16/16638  09:22:04  
Traceback (most recent call last):
  File "/opt/ebsofts/PyTorch/1.6.0-fosscuda-2019b-Python-3.7.4/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 779, in _try_get_data
    data = self._data_queue.get(timeout=timeout)
  File "/opt/ebsofts/Python/3.7.4-GCCcore-8.3.0/lib/python3.7/queue.py", line 179, in get
    self.not_empty.wait(remaining)
  File "/opt/ebsofts/Python/3.7.4-GCCcore-8.3.0/lib/python3.7/threading.py", line 300, in wait
    gotit = waiter.acquire(True, timeout)
  File "/opt/ebsofts/PyTorch/1.6.0-fosscuda-2019b-Python-3.7.4/lib/python3.7/site-packages/torch/utils/data/_utils/signal_handling.py", line 66, in handler
    _error_if_any_worker_fails()
RuntimeError: DataLoader worker (pid 8885) is killed by signal: Killed. 

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/users/g/gabays/.local/bin/ketos", line 10, in <module>
    sys.exit(cli())
  File "/opt/ebsofts/Python/3.7.4-GCCcore-8.3.0/lib/python3.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/opt/ebsofts/Python/3.7.4-GCCcore-8.3.0/lib/python3.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/opt/ebsofts/Python/3.7.4-GCCcore-8.3.0/lib/python3.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/opt/ebsofts/Python/3.7.4-GCCcore-8.3.0/lib/python3.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/ebsofts/Python/3.7.4-GCCcore-8.3.0/lib/python3.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/opt/ebsofts/Python/3.7.4-GCCcore-8.3.0/lib/python3.7/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/home/users/g/gabays/.local/lib/python3.7/site-packages/kraken/ketos.py", line 470, in train
    trainer.run(_print_eval, _draw_progressbar)
  File "/home/users/g/gabays/.local/lib/python3.7/site-packages/kraken/lib/train.py", line 435, in run
    for _, batch in zip(range(self.event_it), self.train_set):
  File "/home/users/g/gabays/.local/lib/python3.7/site-packages/kraken/lib/dataset.py", line 367, in __next__
    sample = next(self.dataset_iter)
  File "/opt/ebsofts/PyTorch/1.6.0-fosscuda-2019b-Python-3.7.4/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 363, in __next__
    data = self._next_data()
  File "/opt/ebsofts/PyTorch/1.6.0-fosscuda-2019b-Python-3.7.4/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 974, in _next_data
    idx, data = self._get_data()
  File "/opt/ebsofts/PyTorch/1.6.0-fosscuda-2019b-Python-3.7.4/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 931, in _get_data
    success, data = self._try_get_data()
  File "/opt/ebsofts/PyTorch/1.6.0-fosscuda-2019b-Python-3.7.4/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 792, in _try_get_data
    raise RuntimeError('DataLoader worker (pid(s) {}) exited unexpectedly'.format(pids_str))
RuntimeError: DataLoader worker (pid(s) 8885) exited unexpectedly
[gabays@gpu006 datasetsOCRSegmenter17]$ 
[gabays@gpu006 datasetsOCRSegmenter17]$ ketos train -t train.txt -e val.txt -f alto
Building training set  [#########################-----------]  11955/16639  00:01:07[182.1345] Text line " " is empty after transformations 
Building training set  [####################################]  16639/16639          
Building validation set  [####################################]  1728/1728          [356.6524] alphabet mismatch: chars in training set only: {'ɔ', 'K', 'k', 'Ç', '–', 'À', 'É', 'æ', 'Ë', 'Ü', '['} (not included in accuracy test during training) 
[356.6526] alphabet mismatch: chars in validation set only: {'Æ'} (not trained) 
Initializing model ✓
stage 1/∞  [------------------------------------]  45/16638  07:32:50  Killed
```
Modules installed are the following:
```
[gabays@gpu006 datasetsOCRSegmenter17]$ module list

Currently Loaded Modules:
  1) GCCcore/8.3.0    12) Python/3.7.4       23) Ninja/1.9.0                          34) x265/3.2             45) Pillow/6.2.1
  2) zlib/1.2.11      13) GCC/8.3.0          24) protobuf/3.10.0                      35) expat/2.2.7          46) cuDNN/7.6.4.38
  3) binutils/2.32    14) CUDA/10.1.243      25) protobuf-python/3.10.0-Python-3.7.4  36) libpng/1.6.37        47) magma/2.5.1
  4) bzip2/1.0.8      15) numactl/2.0.12     26) pybind11/2.4.3-Python-3.7.4          37) freetype/2.10.1      48) NCCL/2.4.8
  5) ncurses/6.1      16) libxml2/2.9.9      27) SciPy-bundle/2019.10-Python-3.7.4    38) util-linux/2.34      49) PyTorch/1.6.0-Python-3.7.4
  6) libreadline/8.0  17) libpciaccess/0.14  28) libyaml/0.2.2                        39) fontconfig/2.13.1    50) Tk/8.6.9
  7) Tcl/8.6.9        18) hwloc/1.11.12      29) PyYAML/5.1.2                         40) X11/20190717         51) Tkinter/3.7.4
  8) SQLite/3.29.0    19) OpenMPI/3.1.4      30) MPFR/4.0.2                           41) FriBidi/1.0.5        52) matplotlib/3.1.1-Python-3.7.4
  9) XZ/5.2.4         20) OpenBLAS/0.3.7     31) NASM/2.14.02                         42) FFmpeg/4.2.1         53) bokeh/1.4.0-Python-3.7.4
 10) GMP/6.1.2        21) FFTW/3.3.8         32) x264/20190925                        43) libjpeg-turbo/2.0.3  54) dask/2.8.0-Python-3.7.4
 11) libffi/3.2.1     22) ScaLAPACK/2.0.2    33) LAME/3.100                           44) LibTIFF/4.0.10       55) scikit-image/0.16.2-Python-3.7.4
```

Could you help me understand what is happening? I am using kraken version 3.0.2. During the installation with pip the following error got raised:

```
ERROR: Could not install packages due to an EnvironmentError: [Errno 13] Permission denied: '/opt/ebsofts/Python/3.7.4-GCCcore-8.3.0/lib/python3.7/site-packages/kraken-3.0.2.dist-info'
Consider using the `--user` option or check the permissions
```

Thanks!

**Comment by gabays** [comment]
It seems to be a problem with

**Comment by mittagessen** [comment]
* Simon Gabay :: 2021-07-16 23:23 Fri:
> Closed #266.

It was a resource limit on the cluster, right?
